### ISSUE TITLE
[change-to-polkit] Cannot toggle any updates items.

### ISSUE THREAD

**Issue by isantop** [pop-os/repoman#27]
After switching over to the PolKit branch, the updates repositories cannot be enabled or disabled. It appears these are being sent over as a Template type, rather than as a string, and Dbus does not know how to handle them.

**Comment by isantop** [comment]
Logging output:

```
ERROR:dbus.connection:Unable to set arguments (<aptsources.distinfo.Template object at 0x7fb3d5791bd0>, True) according to signature None: <class 'TypeError'>: Don't know which D-Bus type to use to encode type "Template"
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/repoman/updates.py", line 165, in on_child_toggled
    self.ppa.set_child_enabled(child, enabled)
  File "/usr/lib/python3/dist-packages/repoman/ppa.py", line 198, in set_child_enabled
    privileged_object.set_child_enabled(child, enabled)
  File "/usr/lib/python3/dist-packages/dbus/proxies.py", line 147, in __call__
    **keywords)
  File "/usr/lib/python3/dist-packages/dbus/connection.py", line 643, in call_blocking
    message.append(signature=signature, *args)
TypeError: Don't know which D-Bus type to use to encode type "Template"
```

**Comment by isantop** [comment]
Fixed as of ac01978
